src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -924,6 +924,61 @@ internal void WriteLine(string format, object arg1)
             }
         }
 
+        internal void WriteLine(string format, string arg1)
+        {
+            WriteLine(format, (object)arg1);
+        }
```

Comment:
Before addressing the comment and comments below.
My initial intention was to create full set of exact overloads to remove overloads with `params`.
Can you confirm that it is right direction?
I believe Roslyn is smart to ignore unused code and we can leave unused overloads so that this problem with extra allocations does not come back.

Reply:
> My initial intention was to create full set of exact overloads to remove overloads with params. Can you confirm that it is right direction?

This is the right direction. But here, what we are addressing is not `params`, but avoiding unnecessary boxing when passing value type to `object` parameter.
Take this overload as an instance, it doesn't add any value, but add one more layer or call redirection, because the existing `WriteLine(string format, object arg1)` already can handle `string format, string arg` input without boxing.

> I believe Roslyn is smart to ignore unused code and we can leave unused overloads so that this problem with extra allocations does not come back.

I'm fine having rest of the overloads that will actually provide a value (avoid boxing), even if they are not used directly yet.

Reply:
I hoped that we could unroll FormatOutputLine() in some way for every specific use.

Fixed - removed.